pyproject.toml

```diff
@@ -11,9 +11,9 @@ version = "0.1.1"
 
 [tool.poetry.dependencies]
 click = "^8.1.3"
-fsspec = "^2023.1.0"
-pydantic = "^1.10.4"
-pygithub = "^1.57"
+fsspec = "^2023.6.0"
+pydantic = ">=1.10.9, <2.0"
```

Comment:
This will cause dependency conflicts with any plugin using `filepattern2` or `bfio>=2.3.0` as both packages now rely on `pydantic>=2.0.0`

Reply:
@Nicholas-Schaub